.github/workflows/backup-kubernetes-databases-inhouse.yml

```diff
@@ -68,7 +68,7 @@ jobs:
 
       - name: Pipelines Version
         run: |
-          echo "Pipelines version: 2.2.2"
+          echo "Pipelines version: 2.3.0"
```

Comment:
Do you think there is a possibility we could extract these numbers directly from repository/release? So, you don't have to always update 21 files with one number?  
If you figure out a way how to pull this info programmatically from the pipeline, feel free to split it into two PRs or something if it feels like it is out of scope of this one.